platforms/documentation/docs/src/docs/userguide/dep-man/declaring-depedencies/dependency_versions.adoc

```diff
@@ -0,0 +1,372 @@
+// Copyright (C) 2023 Gradle, Inc.
+//
+// Licensed under the Creative Commons Attribution-Noncommercial-ShareAlike 4.0 International License.;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://creativecommons.org/licenses/by-nc-sa/4.0/
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[single-version-declarations]]
+= Declaring Versions and Ranges
+
+You can specify dependencies with exact versions or version ranges to define which versions your project can use:
+
+[source,kotlin]
+----
+dependencies {
+    implementation("org.springframework:spring-core:5.3.8")
+    implementation("org.springframework:spring-core:5.3.+")
+    implementation("org.springframework:spring-core:latest.release")
+    implementation("org.springframework:spring-core:[5.2.0, 5.3.8]")
+    implementation("org.springframework:spring-core:[5.2.0,)")
+}
+----
+
+== Understanding version declaration
+
+Gradle supports various ways to declare *versions and ranges*:
+
+[cols="1,1,2"]
+|===
+|Version |Example |Note
+
+|Exact version
+|`1.3`, `1.3.0-beta3`, `1.0-20150201.131010-1`
+|A specific version.
+
+|Maven-style range
+|`[1.0,)`, `[1.1, 2.0)`, `(1.2, 1.5]`
+|`[` `]` indicates inclusive bounds; `(` `)` indicates exclusive bounds.
+
+When the upper or lower bound is missing, the range has no upper or lower bound.
+
+An upper bound exclude acts as a prefix exclude.
+
+|_Prefix_ version range
+|`1.\+`, `1.3.+`
+|Only versions exactly matching the portion before the `\+` are included.
+
+Declaring a version as `+`, without any prefix, will include any version.
+
+|`latest-status` version
+|`latest.integration`, `latest.release`
+|Matches the highest version with the specified status. See link:{javadocPath}/org/gradle/api/artifacts/ComponentMetadata.html#getStatus--[ComponentMetadata.getStatus()].
+
+|Maven `SNAPSHOT` version
+|`1.0-SNAPSHOT`, `1.4.9-beta1-SNAPSHOT`
+|Indicates a snapshot version.
+|===
+
+[[version_ordering]]
+== Understanding version ordering
+
+[source,kotlin]
+----
+dependencies {
+    implementation("org.springframework:spring-core:1.1") // This is a higher version than 1.a
```

Comment:
🤔 I think it would make more sense to talk about "newer" or "later"/"more recent" vs. "higher.

Reply:
Used newer/older.